chempy/electrochemistry/nernst.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function)
+import math
+
+
+def nernst_potential(ion_conc_out, ion_conc_in, charge, T, constants=None, units=None):
+    """
+    Calculates the Nernst potential using the Nernst equation for a particular
+    ion.
+
+    Parameters
+    ----------
+    ion_conc_out: float with unit
+        Extracellular concentration of ion
+    ion_conc_in: float with unit
+        Intracellular concentration of ion
+    charge: integer
+        Charge of the ion
+    T: float with unit
+        Absolute temperature
+    constants: object (optional, default: None)
+        constant attributes accessed:
+            F - Faraday constant
+            R - Ideal Gas constant
+    units: object (optional, default: None)
+        unit attributes: coulomb, joule, kelvin, mol
+
+    Returns
+    -------
+    Membrane potential
+    """
+    if constants is None:
+        F = 96485.33289
+        R = 8.3144598
+        if units is not None:
+            F *= units.coulomb / units.mol
+            R *= units.joule / units.kelvin / units.mol
+    else:
+        F = constants.Faraday_constant
+        R = constants.ideal_gas_constant
+
+    return (R * T) / (charge * F) * math.log(ion_conc_out / ion_conc_in)
```

Comment:
if we add one last kwarg in the function: `backend=math`, and use `backend.log` here, we allow for users to use e.g. SymPy to get a symbolic answer.